sdc/tests/test_series.py

```diff
@@ -4684,6 +4685,68 @@ def test_series_pct_change_impl(S, periods=1, fill_method='pad', limit=None, fre
         msg = 'Method pct_change(). The object periods'
         self.assertIn(msg, str(raises.exception))
 
+    def test_series_setitem_for_value(self):
+        def test_impl(S, val):
+            S[3] = val
+            return S
+
+        hpat_func = self.jit(test_impl)
+        S1 = pd.Series([0, 1, 2, 3, 4])
+        S2 = pd.Series([0, 1, 2, 3, 4])
+        value = 50
+        result_ref = test_impl(S1, value)
+        result = hpat_func(S2, value)
+        pd.testing.assert_series_equal(result_ref, result)
+
+    def test_series_setitem_for_slice(self):
+        def test_impl(S, val):
+            S[2:] = val
+            return S
+
+        hpat_func = self.jit(test_impl)
+        S1 = pd.Series([0, 1, 2, 3, 4])
+        S2 = pd.Series([0, 1, 2, 3, 4])
```

Comment:
I propose to use method `copy()` here, because sometime somebody can change `S1` and forget to change `S2`.
`S2 = S1.copy()`

Reply:
Maybe you need to use parameter `deep`: `S2 = S1.copy(deep=True)`